packages/adapter-drizzle/src/planetscale/index.ts

```diff
@@ -0,0 +1,248 @@
+import {
+  int,
+  timestamp,
+  mysqlTable,
+  varchar,
+  primaryKey,
+} from "drizzle-orm/mysql-core"
+import crypto from 'node:crypto'
+import { drizzle } from "drizzle-orm/planetscale-serverless"
+import { Adapter, AdapterAccount } from "@auth/core/adapters"
+import { connect } from "@planetscale/database"
+import { eq, and } from 'drizzle-orm'
+
+export const users = mysqlTable("users", {
+  id: varchar("id", { length: 255 }).notNull().primaryKey(),
+  name: varchar("name", { length: 255 }),
+  email: varchar("email", { length: 255 }).notNull(),
+  emailVerified: timestamp("emailVerified", { mode: "date" }),
+  image: varchar("image", { length: 255 }),
+})
+
+export const accounts = mysqlTable(
+  "accounts",
+  {
+    userId: varchar("userId", { length: 255 }).notNull(),
+    type: varchar("type", { length: 255 }).$type<AdapterAccount["type"]>().notNull(),
+    provider: varchar("provider", { length: 255 }).notNull(),
+    providerAccountId: varchar("providerAccountId", { length: 255 }).notNull(),
+    refresh_token: varchar("refresh_token", { length: 255 }),
+    access_token: varchar("access_token", { length: 255 }),
+    expires_at: int("expires_at"),
+    token_type: varchar("token_type", { length: 255 }),
+    scope: varchar("scope", { length: 255 }),
+    id_token: varchar("id_token", { length: 255 }),
+    session_state: varchar("session_state", { length: 255 }),
```

Comment:
`linkAccount` function fails due to missing prop in the schema, when trying to register via github:
`refresh_token_expires_in: int("refresh_token_expires_in"),`

Reply:
I got an issue related to linkAccount as well but the issue was that id_token's length was not long enough for some providers. Notably Google and Twitch. I switched it to a Text type which is probably overkill but it works.